### PR TITLE
added initial website

### PR DESCRIPTION
**Pull request by knowbody**
added the initial layout, my CSS skills = none so feel free to do any changes, I thought we should start off with something

Screenshot:
screen shot 2015-11-10 at 20 46 13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,16 +20,19 @@
   ],
   "main": "app.js",
   "scripts": {
-    "start": "gatsby serve"
+    "start": "gatsby serve -p 8080"
```
Comment (timdorr) [inline package.json]: Any reason for the port swap? I don't care either way, just want to be sure I'm not stepping on anyone's toes with the default port choice.

File: app.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 exports.loadContext = function(callback) {
```
Comment (timdorr) [inline app.js]: I think this can be an ES6 style export, right?

Comment (knowbody) [inline app.js]: yup, there are still a few places where I'd like to use ES6 although we can't because of the mixins, old React Router etc.

**Comment by mjackson** [comment]
Just setup a basic HTML page for now (see #7). Thanks though, @knowbody!

**Comment by knowbody** [comment]
@mjackson no problem, this was ages ago. We have the basics now (via #7) so can iterate and improve :) thanks for the write up @mjackson @ryanflorence.

**Pull request closed without merging**